modules/pipeline/providers/leaderworker/config.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package leaderworker
+
+import (
+	"time"
+)
+
+type config struct {
+	Leader leaderConfig `file:"leader"`
+	Worker workerConfig `file:"worker"`
+}
+
+type leaderConfig struct {
+	AlsoBeWorker    bool          `file:"also_be_worker" env:"LEADER_ALSO_BE_WORKER" default:"true"`
```

Comment:
not a good practice, `workerConfig != nil` enough to indicate it's also a worker.

Reply:
no.
only one providers yaml, and workerConfig is a struct and always non-nil. it means leader & worker have same yaml configs.

Reply:
@shuofan suggested me changed to `isWorker` and `LEADER_IS_WORKER`.